config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/glance-operator
-  newTag: v0.0.1
+  newName: quay.io/openstack-k8s-operators/glance-operator-index
```

Comment:
This should actually point to the operator image itself, not the index.  But the tag change is correct.

Reply:
sorry my bad on that one, removing the `-index` while bumping the tag